src/instruments/abstract_instruments/comm/visa_communicator.py

```diff
@@ -72,14 +87,11 @@ def terminator(self):
     def terminator(self, newval):
         if not isinstance(newval, str):
             raise TypeError(
-                "Terminator for VisaCommunicator must be specified "
-                "as a single character string."
-            )
-        if len(newval) > 1:
-            raise ValueError(
-                "Terminator for VisaCommunicator must only be 1 " "character long."
```

Comment:
This is not true, terminator can, e.g., be `"\r\n"`, which in fact is the default for the `write_termination`.

Reply:
thank you, yes this was a very old assumption, and may even be a copy & paste related error